src/MultiInput/MultiInput.js

```diff
@@ -179,6 +189,16 @@ class MultiInput extends Component {
             </List>
         );
 
+        let extendedButtonProps = buttonProps;
```

Comment:
We should keep props unchanged.  Assigning `extendedButtonProps['aria-label'] = buttonLabel;` will also assign `aria-label` to buttonProps.

I'm pretty sure doing this in the component accomplishes the same thing without affecting props.
```jsx
                    <Button
                        aria-label={buttonLabel}
                        {...buttonProps}